src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java

```diff
@@ -193,11 +194,23 @@ private static String getBusBarSectionFieldValue(FieldType field, BusbarSection
         };
     }
 
-    private static String getTerminalFieldValue(FieldType field, Terminal terminal) {
+    private static String getTerminalFieldValue(FieldType field, @Nullable Terminal terminal) {
+        if (terminal == null) {
+            return null;
+        }
         return switch (field) {
             case CONNECTED,
                 CONNECTED_1,
                 CONNECTED_2 -> String.valueOf(terminal.isConnected());
+            case REGULATING_TERMINAL_VL_ID ->
+                    terminal.getVoltageLevel() != null ?
+                    terminal.getVoltageLevel().getId() : null;
+            case REGULATING_TERMINAL_CONNECTABLE_ID ->
+                    terminal.getConnectable() != null ?
+                    terminal.getConnectable().getId() : null;
+            case REGULATION_TYPE -> terminal.getConnectable() != null ?
+                    RegulationType.DISTANT.name() :
+                    RegulationType.LOCAL.name();
```

Comment:
Not sure but I think in the case of a null terminal REGULATION_TYPE should return RegulationType.LOCAL.name(). Isn't it the default mode when a static var compensator has no terminal ?

Reply:
DISTANT => the connectable id must be different to the svar id
The correction is DONE in https://github.com/gridsuite/filter/pull/17/commits/d80d627ef781a2231d9fdda0949f2681506b0c94